resources/assets/components/CampaignSingle/index.js

```diff
@@ -103,23 +105,25 @@ class CampaignSingle extends React.Component {
         {this.props.loading || this.state.loadingNewPosts ?
           <div className="spinner"></div>
         :
-          map(posts, (post, key) =>
-            <Post key={key}
+
+          map(postOrder, function (key, value) {
```

Comment:
Ah, JavaScript strikes again! When using `function(key, value) {` style function declarations, JavaScript creates a new `this` that points to that function (which is rather confusing). If you switch this to use an arrow function `(key, value) => {` you should still be able to access the class's `this` as expected. 😄

Reply:
@DFurnes Ahhh I read that but I was having trouble getting it to work with an arrow function for some reason. I'll give it another shot and shoot up the bat signal if I can't get it :bat: ⚾️

Reply:
Ah! I think it's just because I had a syntax error that I didn't figure out until after I switched it to being a regular `function`.